.ci/controllers-test/machine-class-patch.json

```diff
@@ -0,0 +1,9 @@
+{
+ "providerSpec": {
+  "tags": [
+   "shoot--i544024--sfz-test",
+   "kubernetes-io-cluster-shoot--i544024--sfz-test",
+   "kubernetes-io-role-integration-test"
+  ]
+ }
+}
```

Comment:
this shouldn't be added.. this is added by code itself in case of gcp 
refer https://github.com/gardener/machine-controller-manager/blob/f791f7f3dd372ab1f61bc368b6dad81b8233d132/pkg/test/integration/common/framework.go#L496-L499

Reply:
done